taskcluster/troubleshooting_workers.rst

```diff
@@ -0,0 +1,47 @@
+.. _troubleshooting_workers:
+
+Troubleshooting Workers
+=======================
+
+(This page needs fleshing out. Ideally we have a runbook where we can go from symptom to solution.)
+
+Active workers
+--------------
+
+We can see recent worker activity in the `provisioners UI <https://firefox-ci-tc.services.mozilla.com/provisioners/>`__. For instance, if we're looking at ``gecko-3/b-linux``, we could go `here <https://firefox-ci-tc.services.mozilla.com/provisioners/gecko-3/worker-types/b-linux>`__. Click on the columns to change sorting order. Using the `task started <https://firefox-ci-tc.services.mozilla.com/provisioners/gecko-3/worker-types/b-linux?sortBy=Task%20Started&sortDirection=desc>`__ column can give you an idea of what spot instance workers are still around and running tasks.
+
+Drilling down into the workers themselves, e.g. with `mac-v3-signing2 <https://firefox-ci-tc.services.mozilla.com/provisioners/scriptworker-prov-v1/worker-types/signing-mac-v1/workers/mdc1/mac-v3-signing2>`__, allows you to see the status of recent tasks that ran on that worker. This can give you a better idea if a single worker is busted, or if it's across the pool or multiple pools.
+
+Worker Manager errors
+---------------------
+
+The `worker manager UI <https://firefox-ci-tc.services.mozilla.com/worker-manager>`__ lists the various pools. It shows current capacity and pending tasks, as well as links to view the workers and recent worker manager error messages, for each pool.
+
+Drilling down into each workerType should show you the config. Generally this will be an expanded version of the `ci-configuration worker-pools config <https://hg.mozilla.org/ci/ci-configuration/file/tip/worker-pools.yml>`__.
+
+Common types of error messages
+------------------------------
```

Comment:
Follow-up, not blocking, we should probably enumerate these at some point, if more are expected to be added. But for now it looks good.

Reply:
+1. That's why I added `(This page needs fleshing out. Ideally we have a runbook where we can go from symptom to solution.)` to the top :)